### ISSUE TITLE
Feature request: Folders for each show Recorded TV

### ISSUE THREAD

**Issue by smudger1972** [zipleen/tvheadend-iphone-client#213]
Would it be possible to have folders for each show recorded in the recorded TV section. (Apple TV)
Im currently using MRMC to view my recorded shows as it shows my recordings in folders (this is the way they are organised on my hard drive)
I use TVH client for everyting else. Great App by the way.

**Comment by zipleen** [comment]
Yup I've discussed this two months ago (and that's why you have the folder attribute now showing in the iOS version), but adding a folder architecture to the display will require a complete rewrite of the recordings window.
This will come but only after more important things are fixed / added first :)

**Comment by smudger1972** [comment]
Thanks for that, good to know its part of the plan. 
Using MRMC at the moment for viewing recorded TV is an easy workaround.
keep up the good work

**Comment by mpietruschka** [comment]
Folders are almost the only reason for using MrMC and it's outdated HTS addon (2.2.20 => latest: 7.1.4) on AppleTv. Updating TVHeadend to the latest build (Master / API: 35) makes MrMC fail to connect.

I'd be glad to see folders in TVHClient, too

Thx for your work!

**Comment by zipleen** [comment]
I wanted to do folders with the newest api, but that would probably mean ios13 only! That's why I didn't jump straight away into it yet, but this is something I want to add !
